include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -91,9 +149,22 @@ namespace SuperiorMySqlpp { namespace Converters
             }
         };
 
+        /**
+         * Converts string into uint16_t.
+         * 
+         * @tparam validate Check if string is convertible.
+         */
         template<bool validate>
         struct ToIntegerUnsignedImpl<std::uint16_t, validate>
         {
+            /**
+             * Parses string to uint16_t.
+             * 
+             * @param str String to be converted.
+             * @param length Length of the string.
+             * @throws std::runtime_error exception if length is not between 1 and 5
+             * @return uint16_t integer.
```

Comment:
*integer* - this information is duplicated here because uint16_t contains this information too.
Maybe "Parsed integer value" should be enough in this description. Any thoughts?